jupyterhub/orm.py

```diff
@@ -361,7 +386,7 @@ class Service(Base):
 
     # common user interface:
     name = Column(Unicode(255), unique=True)
-    admin = Column(Boolean, default=False)
+    admin = Column(Boolean(create_constraint=False), default=False)
```

Comment:
Just for my benefit, why is the `(create_constraint=False)` needed here? Isn't it False by default?

Reply:
`create_constraint=False` is the default in sqlalchemy 1.4. This changed from 1.3, so being explicit here means we get consistent behavior on sqlalchemy before and after 1.4.